maps/map06.txt

```diff
@@ -1,3 +1,11 @@
-############
-#o  .     g#
-############
+###########
+#.. ... ..#
+### ### # #
+    ...
+#.# ### #.#
+#.# .o. #.#
+#.# ### #.#
+    ...
+# # ### ###
+#.. ... .g#
+###########
```

Comment:
This level is insane. Two wraparound places makes out BFS AI look incredibly smart! :)